src/main/java/org/broadinstitute/hellbender/utils/clipping/ClippingOp.java

```diff
@@ -39,6 +39,11 @@ public int getLength() {
      * @param originalRead the read to be clipped
      */
     public GATKRead apply(final ClippingRepresentation algorithm, final GATKRead originalRead) {
+        return apply(algorithm, originalRead, true);
+    }
+
+    // Warning, this method is intended for testing purposes
+    GATKRead apply(final ClippingRepresentation algorithm, final GATKRead originalRead, final boolean runAsserts) {
```

Comment:
Just double checking, with the new tests do you still need all these runAsserts changes?